lib/src/components/NxLoadWrapper/__tests__/NxLoadWrapper.test.tsx

```diff
@@ -4,48 +4,77 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
The latest on main changes how `userEvent` should be used.  You should merge from main and then change the imports so that `userEvent` is imported from `../../../__testutils__/rtlUtils` rather than from `@testing-library/user-event`

Reply:
> userEvent

Done 611ef5f7e2320c7c4bc160bdebe144560bacc4ee